UKHomeOffice/cerberus-service
# WIP: COP-12287 - Fixes bug on clearing autocompleted field

## Description
This PR fixes a bug where visually clearing the autocomplete does not clear the form data. The autocomplete in most parts of the airpax & roro forms are now replaced by the multi-select autocomplete (https://ukhomeoffice.github.io/cop-react-components/?path=/docs/d-multiselectautocomplete--default-story) which has also been updated in this PR https://github.com/UKHomeOffice/cop-react-components/pull/140 to support being able to clear the autocomplete value.

## To Test
- Pull & run again dev
- Claim a task, Issue a target
- Observe that once an autocomplete (and not a multi-select autocomplete) is populated, a clear all button `x` is rendered at the end of the autocomplete input field.
- Should the field be mandatory, validation will also kick in when attempting to submitting the form.
